designsafe/apps/api/views.py

```diff
@@ -155,3 +156,41 @@ def post(self, request):
             },
         )
         return HttpResponse("OK", status=202)
+    
+
+
+class SystemMonitorProxyApi(BaseApiView):
+    """"
+    Proxy API for fetching system queue data from Core Portal. 
+    """
+
+    def get(self, request, hostname):
+        CORE_PORTAL_API_BASE = "https://cep.tacc.utexas.edu/api/system-monitor"
```

Comment:
We usually store variables like this in the `common_settings.py` file, so that they can be easily kept track of.  You can see it in the CEP version (we call it `settings.py` there) -- https://github.com/TACC/Core-Portal/blob/c210828226c8c587a76f18e7ae5d8a849adf3a48/server/portal/apps/system_monitor/views.py#L28